include/openPMD/IO/IOTask.hpp

```diff
@@ -638,8 +672,13 @@ struct OPENPMDAPI_EXPORT Parameter<Operation::ADVANCE>
             new Parameter<Operation::ADVANCE>(std::move(*this)));
     }
 
+    struct StepSelection
+    {
+        std::optional<size_t> step;
```

Comment:
please include `<optional>` for stability in this file

Reply:
Done, I've also removed an unused `<map>` import